client/my-sites/site-settings/form-seo.jsx

```diff
@@ -0,0 +1,257 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import Button from 'components/button';
+import SectionHeader from 'components/section-header';
+import ExternalLink from 'components/external-link';
+import Notice from 'components/notice';
+import NoticeAction from 'components/notice/notice-action';
+import protectForm from 'lib/mixins/protect-form';
+import FormInput from 'components/forms/form-text-input-with-affixes';
+import FormInputValidation from 'components/forms/form-input-validation';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+import get from 'lodash/get';
+import notices from 'notices';
+
+export default React.createClass( {
+	displayName: 'SiteSettingsFormSEO',
+
+	mixins: [ protectForm.mixin ],
+
+	getInitialState() {
+		const { site } = this.props;
```

Comment:
@gwwar this code is all `undefined`-safe, so a check here won't actually add anything but noise